common/on-chip-calib.cpp

```diff
@@ -16,16 +16,20 @@
 #include "../src/algo.h"
 #include "../tools/depth-quality/depth-metrics.h"
 
+
+static constexpr float off_value = 0.0f;
+static constexpr float on_value = 1.0f;
+
 namespace rs2
 {
     on_chip_calib_manager::on_chip_calib_manager(viewer_model& viewer, std::shared_ptr<subdevice_model> sub, device_model& model, device dev, std::shared_ptr<subdevice_model> sub_color, bool uvmapping_calib_full)
         : process_manager("On-Chip Calibration"), _model(model), _dev(dev), _sub(sub), _viewer(viewer), _sub_color(sub_color), py_px_only(!uvmapping_calib_full)
     {
-        device_id_string = "Unknown";
+        device_name_string = "Unknown";
         if (dev.supports(RS2_CAMERA_INFO_PRODUCT_ID))
         {
-            device_id_string = _dev.get_info(RS2_CAMERA_INFO_PRODUCT_ID);
-            if (val_in_range(device_id_string, { std::string("0AD3") }))
+            device_name_string = _dev.get_info( RS2_CAMERA_INFO_NAME );
+            if( val_in_range( device_name_string, { std::string( "Intel RealSense D415" ) } ) )
```

Comment:
Why not just come? Why `val_in_range`? (never mind that it's not a range)

Reply:
I guess he align it to the code, it is used this way at several places in the code.